### ISSUE TITLE
Drop `SampleControlInfoAttribute` in favor of `SampleAttribute`

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#16706]
### What would you like to be added

We have two attributes that basically do the same thing. I think it may be time to remove `SampleControlInfoAttribute` and only keep `SampleAttribute`.

To do this, first:

- Replace all attribute usages of `SampleControlInfoAttribute` throughout the pages and user controls in the Samples app with `SampleAttribute`
- And then remove type altogether, keeping only `SampleAttribute`

**Comment by ndwpatrick** [comment]
Hi @Youssef1313 , I'm interested in working on this issue. Can I take it? Also, could you provide any additional details if needed?

**Comment by Youssef1313** [comment]
@MartinZikmund or @jeromelaban should advise here.

**Comment by MartinZikmund** [comment]
@ndwpatrick yes indeed, this is good to take on! The idea here is to find all direct usages of `[SampleControlInfo]` and replace them with `[Sample]` appropriately. Note that the constructors/properties of those two are slightly different, so it is not a plain find and replace, but still quite straightforward conversion.

**Comment by Youssef1313** [comment]
As the number of the usage of the attribute is too high, it may be more viable to create a temporary DiagnosticAnalyzer with CodeFixProvider to do the transition

**Comment by ndwpatrick** [comment]
Thanks for the heads-up @MartinZikmund @Youssef1313. I've created a temporary DiagnosticAnalyzer with CodeFixProvider for this task and along with that I'll have a final manual review myself. If possible can you please assign this to me so that I can start on it.
